person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def of_age?
```

Comment:
- [ ] As recommended for add_rental method in the book.rb file, it applies here too because it would be adding the book and it should do that itself too.